README.md

```diff
@@ -267,12 +267,12 @@ app.dev.gov.uk.		0	IN	A	127.0.0.1
 Provide a local gem path relative to the location of the Gemfile you're editing:
 
 ```ruby
-gem 'govuk_publishing_components', path: '../govuk_publishing_components' 
+gem 'govuk_publishing_components', path: '../govuk_publishing_components'
 ```
 
 ### How to: replicate data locally
 
-There may be times when a full database is required locally.  The following sections give examples of how to replicate this data from integration.  All examples reqire pv, which can be installed on a Mac using Brew (`brew install pv`).
+There may be times when a full database is required locally.  The following sections give examples of how to replicate this data from integration.  All examples require pv, which can be installed on a Mac using Brew (`brew install pv`).
```

Comment:
Rather than adding [more individual `brew install` commands](https://github.com/alphagov/govuk-docker/search?q=brew+install&unscoped_q=brew+install), how about switching to use a [Brewfile](https://github.com/Homebrew/homebrew-bundle#usage)? Then, dependencies can be installed in one go with `brew bundle` at the start of the `govuk-docker setup` script.

Reply:
Is somebody a contributor to Homebrew? @issyl0 🤣

Reply:
I was trying very, very hard to not put that hat on while still being useful. :joy:

Reply:
Also sorry, the diff at a glance made it look like you added the entire line when you didn't. I'll tackle all this stuff separately I guess. :-)

Reply:
@issyl0 I like your suggestion, but can I do it in a separate PR with the change to the setup script?